modules/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -200,6 +200,15 @@ impl TryFrom<RawClientState> for ClientState {
             .clone()
             .ok_or_else(Error::missing_trusting_period)?;
 
+        let frozen_height = raw.frozen_height.and_then(|raw_height| {
```

Comment:
Raw frozen height is defined as an `Option` [here](https://github.com/informalsystems/ibc-rs/blob/master/proto/src/prost/ibc.lightclients.tendermint.v1.rs#L21) ->
```rust
/// Block height when the client was frozen due to a misbehaviour
#[prost(message, optional, tag = "6")]
pub frozen_height: ::core::option::Option<super::super::super::core::client::v1::Height>,
```
I would've expected gaiad to respond with a `None` instead of `RawHeight::zero()` to indicate a client that isn't frozen. Seems to me like a bug in ibc-go(?), although I agree we need this fix for now anyway.